src/main/java/org/sagebionetworks/bridge/spring/controllers/CRCController.java

```diff
@@ -181,6 +203,129 @@ String getUserAgent() {
         return (userAgent == null) ? "<Unknown>" : userAgent;
     }
 
+    @PostMapping("v1/cuicm/participants/self/labshipments/request")
+    public ResponseEntity<StatusMessage> postUserLabShipmentRequest() {
+        // caller enrolled studies
+        UserSession session = getAuthenticatedSession();
+        App app = appService.getApp(session.getAppId());
+        RequestContext requestContext = RequestContext.get();
+        Set<String> callerEnrolledStudies = requestContext.getCallerEnrolledStudies();
+        Set<String> orgSponsoredStudies = requestContext.getOrgSponsoredStudies();
+        RequestContext.set(requestContext.toBuilder().withOrgSponsoredStudies(callerEnrolledStudies)
+                .withCallerEnrolledStudies(
+                        orgSponsoredStudies).build());
```

Comment:
?! I would rather not swap these as that seems like it will be *really* confusing...how about setting them to nulls or empty sets?

Reply:
Participant self call does not have OrgSponsoredStudies

Admin call for a participant does have OrgSponsoredStudies. I remove this so the report's studies match when called from either endpoint